common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.commercetools.sunrise.common;
+
+import io.sphere.sdk.client.SolutionInfo;
+import io.sphere.sdk.meta.BuildInfo;
+
+public class SunriseSolutionInfo extends SolutionInfo {
+    public SunriseSolutionInfo() {
+        setName("");
+        setVersion(BuildInfo.version());
```

Comment:
Have you checked this yields the correct build info for Sunrise? This seems to be coming from the JVM SDK.